protocol/dydx-cosmwasm/src/proto_structs.rs

```diff
@@ -3,9 +3,142 @@ use serde::{Deserialize, Serialize};
 
 use cosmwasm_std::Uint64;
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
```

Comment:
Per offline discussion let's a TODO item to on how to standardize proto compilation for rust, and also cover this in the detailed spec.